Doc/library/ensurepip.rst

```diff
@@ -122,6 +123,12 @@ Module API
    *verbosity* controls the level of output to :data:`sys.stdout` from the
    bootstrapping operation.
 
+   *prefix* specifies the directory prefix to use when installing.
```

Comment:
I think we should explain the difference between *root* and *prefix* better. Per now, it is not obvious how they differ.

Reply:
I agree it's not obvious. And that's speaking as a pip maintainer... It's all shrouded in mystery and legacy behaviour dating back to the origins of distutils, I think. And it's very much to do with Unix - on Windows, those options are essentially never used (which is mostly why I don't understand them...)

But I think it's fair to assume that anyone using these options is doing so because they know of the same options in pip, and so they understand what they are doing (or at least their confusion isn't our problem). It's not ideal, but OTOH, this isn't the place for a tutorial on packaging concepts, either.

Reply:
root vs prefix goes back to the base unix model I think, and may be explained in make documentation or something similarly old.

Reply:
DESTDIR is the make equivalent of specifying --root